README.md

```diff
@@ -373,3 +373,18 @@ OPEN_DISCUSSIONS_BASE_URL=http://docker.for.mac.localhost:8063/
 - You should be redirected back to the open-discussions home page, and be logged in.
 - Log out & back in as a superuser and to go the Users admin page.
   - There should be a new user with the same email address and name that you used to register with SSOCircle.
+
+
+## Commits
+
+To ensure commits to github are safe, you should install the following first:
+```
+pip install pre_commit
```

Comment:
Should this be `pip install pre_commit detect-secrets`

Reply:
Turns out that part isn't necessary, it will be installed by precommit based on the configuration when the first commit is attempted:
```
[INFO] Initializing environment for git@github.com:Yelp/detect-secrets.
[INFO] Installing environment for git@github.com:Yelp/detect-secrets.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
```